pkg/pcr/replay.go

```diff
@@ -26,10 +26,18 @@ func Replay(eventLog *tpmeventlog.TPMEventLog, pcrIndex pcr.ID, hashAlgo tpmeven
 	}
 
 	var result []byte
+	switch pcrIndex {
+	case 0:
+	case 1:
```

Comment:
Add a comment on what exactly we're doing in this block would be good. It seems basically we assume PCR0, and if it's PCR1, we adjust what our expected result size should be.

Reply:
Fixed.